src/panels/media-browser/ha-panel-media-browser.ts

```diff
@@ -69,15 +78,25 @@ class PanelMediaBrowser extends LitElement {
                     .narrow=${this.narrow}
                   ></ha-menu-button>
                 `}
-            <div main-title class="heading">
-              <div>
-                ${!this._currentItem
-                  ? this.hass.localize(
-                      "ui.components.media-browser.media-player-browser"
-                    )
-                  : this._currentItem.title}
-              </div>
+            <div main-title>
+              ${!this._currentItem
+                ? this.hass.localize(
+                    "ui.components.media-browser.media-player-browser"
+                  )
+                : this._currentItem.title}
             </div>
+            ${isLocalMediaSourceContentId(
+              this._navigateIds[this._navigateIds.length - 1]
+                .media_content_id || ""
+            )
```

Comment:
This doesn't allow uploading directly to the media/ folder

Only to folders within that

Reply:
Ah yes, because we don't use the item that the navigation ID actually resolved to. Let me fix it.

Reply:
Fixed.

I didn't catch it because I have multiple media sources, so it won't automatically go to the first folder.